src/sql/azdata.proposed.d.ts

```diff
@@ -414,7 +414,7 @@ declare module 'azdata' {
 		/**
 		 * The cell value
 		 */
-		value: string | number | boolean | Component | DeclarativeTableMenuCellValue;
+		value: string | number | boolean | Component | DeclarativeTableMenuCellValue | Date;
```

Comment:
Why not just convert it to a string first instead?

Reply:
yes can change, i didnt want to manipulate date, may be for display i can or may in the map too. But just not sure if there is a better way to convert string to Date ( in iso-8601 ) format 
from date to ISO string is easy. Still figuring out ..

Reply:
Let's go with just using a string for now - in the future I think it would be useful to support dates (to help ensure that dates are displayed consistently across the app) but that should be a separate work item if we decide it's actually a needed feature.

@alanrenmsft FYI